core/src/main/kotlin/com/github/quillraven/darkmatter/ui/ConfirmDialog.kt

```diff
@@ -0,0 +1,38 @@
+package com.github.quillraven.darkmatter.ui
+
+import com.badlogic.gdx.scenes.scene2d.ui.Dialog
+import com.badlogic.gdx.scenes.scene2d.ui.Label
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton
+import com.badlogic.gdx.utils.Align
+import com.badlogic.gdx.utils.I18NBundle
+import com.github.quillraven.darkmatter.V_HEIGHT_PIXELS
+import com.github.quillraven.darkmatter.V_WIDTH_PIXELS
+import ktx.scene2d.Scene2DSkin
+
+private const val ELEMENT_PADDING = 7f
+private const val DIALOG_WIDTH_SCALE = 0.75f
+private const val DIALOG_HEIGHT_SCALE = 0.5f
+
+class ConfirmDialog(bundle: I18NBundle) : Dialog("", Scene2DSkin.defaultSkin, SkinWindow.DEFAULT.name) {
+    val yesButton = TextButton(" ${bundle["yes"]} ", Scene2DSkin.defaultSkin, SkinTextButton.DEFAULT.name)
```

Comment:
Is there a reason why you're wrapping the text with spaces? You should be able to define paddings for each button individually, if that's the problem.

Reply:
The reason is that the text gets inside the background graphic of the button because how should the text know that the graphic is a frame with some borders? A space seems to be the simplest solution to get the text inside the graphic frame.

Otherwise, I would need to set the padding of the label inside the button or something like that and I did not check if that is possible. For the time being, the space does the job ;)

![image](https://user-images.githubusercontent.com/93260/80282742-7c6f0380-8713-11ea-999d-e96b3b03efc6.png)

Reply:
I think would should be able to fix that with `button.pad` or by leaving extra space in your ninepatch. Just as in word documents, you generally shouldn't control the layout with whitespaces. :)

Reply:
found a way

```kotlin
val noButton = scene2d.textButton(bundle["no"], SkinTextButton.DEFAULT.name) {
        labelCell.padLeft(ELEMENT_PADDING).padRight(ELEMENT_PADDING)
    }
```